ACE/ace/OS_NS_signal.inl

```diff
@@ -15,6 +15,14 @@ kill (pid_t pid, int signum)
   ACE_UNUSED_ARG (pid);
   ACE_UNUSED_ARG (signum);
   ACE_NOTSUP_RETURN (-1);
+#elif defined (ACE_VXWORKS)
+  /*
```

Comment:
What when RTP is used instead of kernel mode, at that moment VxWorks does have support for a process